pkg/bot-code/client.go

```diff
@@ -0,0 +1,289 @@
+package botgithub
```

Comment:
Remove this file entirely.
Apply the requested changes to the file at `pkg/bot-github/client.go`